components/hab/src/cli.rs

```diff
@@ -172,16 +176,7 @@ pub fn get() -> App<'static, 'static> {
             (about: "Commands relating to Habitat packages")
             (aliases: &["p", "pk", "package"])
             (@setting ArgRequiredElseHelp)
-            (@subcommand binlink =>
```

Comment:
Why the switch from macro -> builder pattern? We prefer the macro syntax and only fallback to builder when we need it to dynamically generate things or load CLI bits from other crates